tensorflow/python/pywrap_tfe.i

```diff
@@ -106,19 +106,19 @@ limitations under the License.
 }
 
 %typemap(in) const char* serialized_function_def {
-  $1 = TFE_GetPythonString($input);
+  $1 = const_cast<char*>(TFE_GetPythonString($input));
```

Comment:
This is certainly unfortunate.
Apparently, SWIG intentionally ignores the 'const'-ness of the parameter. Let's add a comment to that effect in all these 3 sites:

// For const parameters in a function, SWIG pretty much ignores the const.
// See: http://www.swig.org/Doc2.0/SWIG.html#SWIG_nn13
// Hence the 'const_cast'.